### DISCUSSION TITLE
数学家教辅导/作业辅导

### DISCUSSION THREAD

**Discussion by JinLingxi** [share121/inter-knot#2917] [其他]
新艾利都最佳辅导绳匠！！！
![MathCyclus](https://github.com/user-attachments/assets/27ac3b04-bc5c-4ad9-bec6-ce75c8ca945c)
目前双一流数学系硕士在读
可以辅导高考培优以及本科范围内的数学课程！！
楼主还有数学建模和数学竞赛经历，有相关需求的市民可以联系楼主呀~

**Comment by I-ikun** [comment]
不是哥们，当绳匠这么卷吗？<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by kbchino** [comment]
这这这，太有实力了吧
